src/components/Forms/Form.js

```diff
@@ -67,7 +70,8 @@ class Form extends React.Component {
 
   formIsValid() {
     const components = Object.values(this.formComponents)
-    return components.every( component => component.validate() )
```

Comment:
@nbwar The reason why I make the change here is because `Array.prototype.every()` immediately returns `false` if found. This can lead to a somewhat confusing UX like below ⬇️
<img width="537" alt="screen shot 2018-10-16 at 10 42 48 pm" src="https://user-images.githubusercontent.com/3990811/47064624-1526d980-d195-11e8-9e3e-113eb177026d.png">
Last name is actually valid here but red because it failed on a prior submit. 
Let me know you think :)